internal/cmd/diffs.go

```diff
@@ -35,8 +36,9 @@ import (
 )
 
 const (
```

Comment:
Could you please add an regex101.com example for these regex strings? This would make it easier to understand how they match :)